appinfo/info.xml

```diff
@@ -8,9 +8,14 @@
 	<dependencies>
 		<owncloud min-version="9.0" max-version="9.1" />
 	</dependencies>
+
```

Comment:
Had no intention. You can kick it.


Am 2016-06-02 um 18:24 schrieb Georg Ehrke:
>
> In appinfo/info.xml 
> <https://github.com/owncloud/calendar/pull/564#discussion_r65572419>:
>
> > @@ -8,9 +8,14 @@
> >  	<dependencies>
> >  		<owncloud min-version="9.0" max-version="9.1" />
> >  	</dependencies>
> > +
>
> Why did you add this empty line?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/owncloud/calendar/pull/564/files/3f50ff3217d392ced984e6d5dc00210a056b3d13#r65572419>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ALEnw9GxCIS89_xxeJ2-gdxXmq11lNnnks5qHwPIgaJpZM4Isqi9>.
>

-- 
Jan Klostermann
- The Agile Management Sounding-Board -
Systems Engineer & Mediator
Am Isarkanal 24/15
81379 München

Tel.: +49-89/74563382
Mobil: +49-179/4628819
Sip: +49-89/21551663 <sip:08921551663@sipgate.de>
Skype: jan.klostermann <callto://jan.klostermann/>
Threema: P58VSJUY <threema://add?id=P58VSJUY>
Email: jk@janklostermann.de <mailto:jk@janklostermann.de>
Netzwerke: Xing <https://www.xing.com/profile/Jan_Klostermann>, LinkedIn 
<http://de.linkedin.com/in/janklostermann>